mpas_analysis/ocean/ocean_regional_profiles.py

```diff
@@ -331,6 +360,11 @@ def run_task(self):
                                     'data': np.arange(nVertLevels)})
 
         vertMask = vertIndex < dsRestart.maxLevelCell
+        if self.max_bottom_depth:
+            depthMask = dsRestart.bottomDepth < self.max_bottom_depth
+            vertDepthMask = vertMask * depthMask
```

Comment:
```suggestion
            vertDepthMask = np.logical_and(vertMask, depthMask)
```
If either were float arrays, the multiplication would be fine but since both are bool arrays, it's better to use the `logical_and()`.